aeneas/src/vst/Vst.v3

```diff
@@ -375,9 +375,16 @@ class SuperClause {
 	new(point, args) { }
 }
 
-class VstRepHint {
-	def list: List<VstIdent<void>>;
-	new(list) { }
+type VstRepHint {
+	case Boxed;
+	case Unboxed;
+	case Packing(p: VstPacking);
+	case Other(hint: string);
+}
+
+type VstPacking {
+	case Bits(rep: string);
+	case Params(packingName: Token, fields: VstList<Token>);
```

Comment:
What about making the fields a `VstList<Expr>`? That way we can accommodate `VarExpr` and `Literal`.